### PR TITLE
update changelog kube-proxy distroless based on feedback

### PR DESCRIPTION
**Pull request by csantanapr**
update changelog based on [feedback](https://github.com/kubernetes/kubernetes/pull/111983/files#r955452389)

/kind cleanup
/kind documentation
/sig release

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.25.md [hunk 1]
```diff
@@ -278,24 +278,24 @@ Introduce KMS v2alpha1 API to add performance, rotation, and observability impro
 
 ### Kube-proxy images are now based in distroless
```
Comment (dims) [inline CHANGELOG/CHANGELOG-1.25.md:279]:
```suggestion
### Kube-proxy images are now based on distroless images
```

**Comment by cici37** [comment]
/lgtm
Thank you!

**Comment by puerco** [comment]
@csantanapr can you cherry-pick this to the `release-1.25` branch to even the documents? :pray:

**Pull request merged**